utils/cr_demo/demo_utils.py

```diff
@@ -0,0 +1,396 @@
+    """Stop servers.
+
+    Args:
+        force (bool): Whether to use --force. Defaults to None.
+    """
+    system_stop_cmd = ["dmg", "system", "stop"]
+    if force:
+        system_stop_cmd.append("--force")
+    run_command(command=system_stop_cmd)
+
+
+def system_start():
+    """Start servers."""
+    system_start_cmd = ["dmg", "system", "start"]
+    run_command(command=system_start_cmd)
+
+
+def system_query(json=False, verbose=False):
+    """Call dmg system query
+
+    Args:
+        json (bool): Whether to use --json. Defaults to False.
+        verbose (bool): Whether to use --verbose. Defaults to False.
+
+    Returns:
+        str: Command output.
+
+    """
+    if json:
+        system_query_cmd = ["dmg", "--json", "system", "query"]
+    else:
+        system_query_cmd = ["dmg", "system", "query"]
+    if verbose:
+        system_query_cmd.append("--verbose")
+    command = " ".join(system_query_cmd)
+    print(f"Command: {command}")
+
+    if json:
+        result = subprocess.run(
+            system_query_cmd, stdout=subprocess.PIPE, universal_newlines=True,
+            check=False)
+        return result.stdout
+
+    subprocess.run(system_query_cmd, check=False)
+    return None
+
+
+# Utility methods
+def create_uuid_to_seqnum():
+    """Create pool UUID to sequence number mapping.
+
+    Returns:
+        dict: UUID to sequence number mapping for each pool. Sequence number will be used
+            during repair.
+
+    """
+    uuid_to_seqnum = {}
+    stdout = check_query(json=True)
+    generated_yaml = yaml.safe_load(stdout)
+    for report in generated_yaml["response"]["reports"]:
+        uuid_to_seqnum[report["pool_uuid"]] = report["seq"]
+
+    return uuid_to_seqnum
+
+
+def create_label_to_uuid():
+    """Create label to UUID mapping.
+
+    Returns:
+        dict: Pool label to UUID.
+
+    """
+    label_to_uuid = {}
+    stdout = list_pool(json=True)
+    generated_yaml = yaml.safe_load(stdout)
+    for pool in generated_yaml["response"]["pools"]:
+        label_to_uuid[pool["label"]] = pool["uuid"]
+
+    return label_to_uuid
+
+
+def get_current_labels():
+    """Get current pool labels from MS.
+
+    Returns:
+        list: Current pool labels.
+
+    """
+    pool_labels = []
+    stdout = list_pool(json=True)
+    generated_yaml = yaml.safe_load(stdout)
+    for pool in generated_yaml["response"]["pools"]:
+        pool_labels.append(pool["label"])
+
+    return pool_labels
+
+
+def convert_list_to_str(original_list, separator):
+    """Convert given list to a string with each item separated by separator.
+
+    Args:
+        original_list (list): List of items.
+        separator (str): Separator to separate each item in the new string list.
+
+    Returns:
+        str: String list.
+
+    """
+    str_list = ""
+    for item in original_list:
+        if str_list == "":
+            str_list = str(item)
+        else:
+            str_list += separator + str(item)
+
+    return str_list
```

Comment:
Optional
FYI - It looks like you just want to convert each item to a string and join, right? This works too
```
return separator.join(map(str, original_list))
```

Reply:
Neat. Thanks.